en/advanced_config/esc_calibration.md

```diff
@@ -24,33 +48,92 @@ To calibrate the ESCs:
-In *theory* this means that the outputs might be can be configured by setting the values provided in the ESC technical specification for each output during [Actuator Configuration](../config/actuators.md) (under the hood this sets the [PWM_MAIN_MINn](../advanced_config/parameter_reference.md#PWM_MAIN_MIN1)/[PWM_AUX_MINn](../advanced_config/parameter_reference.md#PWM_AUX_MIN1) and [PWM_MAIN_MAXn](../advanced_config/parameter_reference.md#PWM_MAIN_MAX1)/[PWM_AUX_MAXn](../advanced_config/parameter_reference.md#PWM_AUX_MAX1) parameters).
+   Following ESC calibration all motors with the same (re)calibrated ESCs should behave in the same way for the same inputs, and the default PWM settings for motor outputs in the actuator configuration should work out of the box.
+   
+   You should confirm that the motors do indeed work correctly.
+   Since the default configuration values have been set conservatively, you may also wish to tune them for your particular ESCs.
+   
+   :::note
+   The steps below are similar to those described in [Actuator Configuration > Motor Configuration](../config/actuators.md#motor-configuration).
+   The instructions here are optimized for the assumption that the ESCs have been calibrated.
+   :::
 
-In practice the input range may differ even on high quality controllers.
-Using this calibration tool is recommended, as it ensures that all ESC behave exactly the same way.
-:::
+   Verify the following values:
+
+   - The minimum value for a motor (default: `1100us`) should make the motor spin slowly but reliably, and also spin up reliably after it was stopped.
+
+     You can confirm that a motor spins at minimum (still without propellers) in [Actuator Testing](../config/actuators.md#actuator-testing), by enabling the sliders, and then moving the test output slider for the motor to the first snap position from the bottom.
+     The correct value should make the motor spin immediately and reliably as you move the slider from disarmed to minimum.
+
+     To find the "optimal" minimum value, move the slider to the bottom (disarmed).
+     Then increase the PWM output's `disarmed` setting in small increments (e.g. 1025us, 1050us, etc), until the motor starts to spin reliably (it is better to be a little too high than a little too low).
+     Enter this value into the `minimum` setting for all the motor PWM outputs, and restore the `disarmed` output to `1100us`.
+
+   - The maximum value for a motor (default: `1900us`) should be chosen such that increasing the value doesn't make the motor spin any faster.
+
+     You can confirm that the motor spins quickly at the maximum setting in [Actuator Testing](../config/actuators.md#actuator-testing), by moving the associated test output slider to the top snap position.
+
+     To find the "optimal" maximum value, first move the slider to the bottom (disarmed).
+     Then increase the PWM output's `disarmed` setting to near the default maximum (`1900`) - the motors should spin up.

```

Comment:
Should we suggest a lower value as starting point - i.e. is it possible that the 1900 value will already be too high?

Reply:
I will assume not.